google-cloud-spanner/src/test/java/com/google/cloud/spanner/ValueBinderTest.java

```diff
@@ -239,6 +262,10 @@ public static Date defaultDate() {
       return Arrays.asList("a", "b");
     }
 
+    public static Iterable<String> defaultJsonIterable() {
```

Comment:
I don't think so, but according to IntelliJ most of the methods aren't being used anywhere (all the ones in grey):

![image](https://user-images.githubusercontent.com/8233863/129648295-ffe9403e-e4e3-4829-bd34-386c770d338a.png)

so added this just in case. Maybe we can remove these in another cleanup PR?